# Traffic update describes a vehicle with id v_id being spotted at
# node n_id. The traffic manager will handle this information 
# appropriately

int32 v_id
int32 n_id